packages/mask/src/plugins/Trader/SNSAdaptor/trending/TrendingViewDeck.tsx

```diff
@@ -338,15 +336,14 @@ export function TrendingViewDeck(props: TrendingViewDeckProps) {
                                             lineHeight="24px"
                                             color={theme.palette.maskColor.dark}>
                                             {isNFT ? `${t('plugin_trader_floor_price')}: ` : null}
-                                            <FormattedCurrency
-                                                value={
-                                                    (trending.dataProvider === SourceType.CoinMarketCap
-                                                        ? last(stats)?.[1] ?? market.current_price
-                                                        : market.current_price) ?? 0
-                                                }
-                                                sign={isNFT ? market.price_symbol : 'USD'}
-                                                formatter={formatCurrency}
-                                            />
+                                            {formatCurrency(
```

Comment:
Any difference?

Reply:
The `FormatCurrency` doesn't pass `options` to `formatCurrency`